keyboards/kinesis/kintwin/info.json

```diff
@@ -0,0 +1,239 @@
+{
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "board": "BLACKPILL_STM32_F401",
+    "keyboard_name": "Kinesis",
+    "url": "https://github.com/alvicstep/kintwin",
+    "manufacturer": "https://github.com/alvicstep/kintwin",
+    "maintainer": "Alex Stepanov",
+    "usb": {
+        "device_version": "0.0.3",
+        "vid": "0xFEED",
+        "pid": "0x4B4E"        
+    },
+    "matrix_pins": {
+        "rows": ["A0", "A1", "A2", "A3", "A4", "A5", "A6"],
+        "cols": ["A7", "B0", "B1", "B10", "B9", "B8", "B7", "B14", "B13", "B12", "C14", "C15", "B15", "B6", "B5"]
+    },
+    "diode_direction": "ROW2COL",
+    "indicators": {
+        "caps_lock": "A15",
+        "scroll_lock": "B3",
+        "on_state": 0
```

Comment:
looking at similar kinesis conversions, is `num_lock` missing here?

Reply:
will add both num_lock and compose